src/fp_liste.cpp

```diff
@@ -70,7 +70,7 @@ void Liste::add(double X_, double lthrs_, double rthrs_, double lslope, double r
 	//cout << "Border : " << lBorder << "," << rBorder << "then" << Current->cFunc->lBorder << "," << Current->cFunc->rBorder << "\n";
 	//}
 
-        if(Current->cFunc->rBorder != lBorder & Current->cFunc->lBorder < lBorder){
+        if((Current->cFunc->rBorder != lBorder) && (Current->cFunc->lBorder < lBorder)){
```

Comment:
& means the same thing as && in C++ but && is preferred for clarity https://stackoverflow.com/questions/24542/using-bitwise-operators-for-booleans-in-c